src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props

```diff
@@ -1,7 +1,6 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>5.0.1</PackageVersion>
-    <AssemblyVersion>5.0.0.1</AssemblyVersion>
```

Comment:
Why is this showing as a diff here?  We can't go backwards in AssemblyVersion.  Was this shipped this way?  I thought you had reverted the AssemblyVersions in the shipping package.

Reply:
We were late in the cycle so we decided to fix this in 5.0.2

Reply:
@Anipik just to double check, do you intend to ship the 5.0.2 version of this package with 5.0.0.0 as the assembly version even though the 5.0.1 version of the package had an assembly version of 5.0.0.1? Sounds concerning to me.

Reply:
Confirmed that this shipped with the incremented version, we have to keep it:
```
// C:\Users\erics\AppData\Local\Temp\Temp1_microsoft.extensions.dependencyinjection.5.0.1.nupkg\lib\net5.0\Microsoft.Extensions.DependencyInjection.dll
// Microsoft.Extensions.DependencyInjection, Version=5.0.0.1, Culture=neutral, PublicKeyToken=adb9793829ddae60
```